lib/core/card-view/components/card-view-item-dispatcher/card-view-item-dispatcher.component.ts

```diff
@@ -92,6 +98,8 @@ export class CardViewItemDispatcherComponent implements OnChanges {
         this.componentReference.instance.editable = this.editable;
         this.componentReference.instance.property = this.property;
         this.componentReference.instance.displayEmpty = this.displayEmpty;
+        this.componentReference.instance.displayNoneOption = this.displayNoneOption;
```

Comment:
Please add these new fields for testing inside this test suite:
`should update the subcomponent\'s input parameters`

Reply:
done